apps/api/src/server/app/state-machine/__tests__/full-planning-appeal.machine.test.js

```diff
@@ -21,17 +21,23 @@ test.before('sets up mocking of actions', () => {
 });
 
 /**
- * @param {object} t unit test
- * @param {string} initialState initial state in state machine
- * @param {string} action action taken to proceed in state machine
- * @param {string} expectedState expected state after action was taken
- * @param {object} context Context of transition
- * @param {boolean} hasChanged True if action was valid, False if action was invalid
+ * @param {object} t
```

Comment:
This declaration is incorrect, but it's a jsdoc thing anyway which is an upcoming ticket we can address this in

Reply:
To be solved in BOAS-17